cdap-app-fabric/src/test/java/co/cask/cdap/gateway/handlers/AuthorizationHandlerTest.java

```diff
@@ -136,29 +136,72 @@ public Authorizer get() {
             .build())
         .build());
 
-    NamespaceId ns1 = Ids.namespace("ns1");
-    Principal admin = new Principal("admin", Principal.PrincipalType.USER);
-    try {
-      client.authorized(ns1, admin, Action.READ);
-      Assert.fail();
-    } catch (FeatureDisabledException expected) {
-      Assert.assertEquals("Authorization", expected.getFeature());
-    }
-
-    try {
-      client.grant(ns1, admin, ImmutableSet.of(Action.READ));
-      Assert.fail();
-    } catch (FeatureDisabledException expected) {
-      Assert.assertEquals(Constants.Security.Authorization.ENABLED, expected.getEnableConfigKey());
-      Assert.assertEquals("true", expected.getEnableConfigValue());
-    }
-
-    try {
-      client.revoke(ns1, admin, ImmutableSet.of(Action.READ));
-      Assert.fail();
-    } catch (FeatureDisabledException expected) {
-      Assert.assertEquals("cdap-site.xml", expected.getConfigFile());
-    }
+    final NamespaceId ns1 = Ids.namespace("ns1");
+    final Principal admin = new Principal("admin", Principal.PrincipalType.USER);
+    final Role admins = new Role("admins");
+
+    verifyFeatureDisabled(new DisabledFeatureCaller() {
```

Comment:
Can you explain a bit about what these verifyFeatureDisabled calls are doing, and may be a leave a comment too?